Source/WebCore/accessibility/AccessibilityRenderObject.cpp

```diff
@@ -2456,7 +2456,7 @@ void AccessibilityRenderObject::addNodeOnlyChildren()
             continue;
         }
 
-        if (!nodeHasDisplayContents(*child) && !isNodeAriaVisible(*child))
+        if (!nodeHasDisplayContents(*child) || isAXHidden())
```

Comment:
Based on our discussion in-person, I think we don't need this `|| isAXHidden()` check?